member-service/src/main/java/com/hedvig/memberservice/aggregates/MemberAggregate.kt

```diff
@@ -0,0 +1,588 @@
+import com.hedvig.memberservice.events.DanishSSNUpdatedEvent
+import com.hedvig.memberservice.events.EmailUpdatedEvent
+import com.hedvig.memberservice.events.FraudulentStatusUpdatedEvent
+import com.hedvig.memberservice.events.InsuranceCancellationEvent
+import com.hedvig.memberservice.events.LivingAddressUpdatedEvent
+import com.hedvig.memberservice.events.MemberAuthenticatedEvent
+import com.hedvig.memberservice.events.MemberCancellationEvent
+import com.hedvig.memberservice.events.MemberCreatedEvent
+import com.hedvig.memberservice.events.MemberInactivatedEvent
+import com.hedvig.memberservice.events.MemberSignedEvent
+import com.hedvig.memberservice.events.MemberSignedWithoutBankId
+import com.hedvig.memberservice.events.MemberSimpleSignedEvent
+import com.hedvig.memberservice.events.MemberStartedOnBoardingEvent
+import com.hedvig.memberservice.events.NameUpdatedEvent
+import com.hedvig.memberservice.events.NewCashbackSelectedEvent
+import com.hedvig.memberservice.events.NorwegianMemberSignedEvent
+import com.hedvig.memberservice.events.NorwegianSSNUpdatedEvent
+import com.hedvig.memberservice.events.OnboardingStartedWithSSNEvent
+import com.hedvig.memberservice.events.PersonInformationFromBisnodeEvent
+import com.hedvig.memberservice.events.PhoneNumberUpdatedEvent
+import com.hedvig.memberservice.events.PickedLocaleUpdatedEvent
+import com.hedvig.memberservice.events.SSNUpdatedEvent
+import com.hedvig.memberservice.events.TrackingIdCreatedEvent
+import com.hedvig.memberservice.services.cashback.CashbackService
+import com.hedvig.memberservice.util.SsnUtilImpl.Companion.getBirthdateFromSwedishSsn
+import com.hedvig.memberservice.util.logger
+import com.hedvig.memberservice.web.dto.Nationality.Companion.toMemberSimpleSignedEventNationality
+import com.hedvig.memberservice.web.dto.Nationality.Companion.toSSNUpdatedEventNationality
+import org.axonframework.commandhandling.CommandHandler
+import org.axonframework.commandhandling.model.AggregateIdentifier
+import org.axonframework.commandhandling.model.ApplyMore
+import org.axonframework.eventhandling.EventHandler
+import org.axonframework.eventsourcing.EventSourcingHandler
+import org.axonframework.messaging.MetaData
+import org.axonframework.spring.stereotype.Aggregate
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.lang.Nullable
+import java.io.IOException
+import java.util.UUID
+import java.util.function.Supplier
+
+@Aggregate
+class MemberAggregate() {
+
+    @AggregateIdentifier
+    private var id: Long = 0L
+
+    @Autowired
+    lateinit var bisnodeClient: BisnodeClient
+
+    @Autowired
+    lateinit var cashbackService: CashbackService
+
+    @Autowired
+    lateinit var objectMapper: ObjectMapper
+
+    @Autowired
+    lateinit var uuidGenerator: UUIDGenerator
+
+    lateinit var status: MemberStatus
+    lateinit var member: Member
+    var trackingId: UUID? = null
+    lateinit var latestBisnodeInformation: BisnodeInformation
```

Comment:
Since this is not set in the "MemberCreatedEvent" I'd set it to null instead
```suggestion
   var latestBisnodeInformation: BisnodeInformation? = null
```